### ISSUE TITLE
Consider using 64M1 microcontrollers

### ISSUE THREAD

**Issue by brunofalmeida** [HeronMkII/pcbs#65]
We might want to use the 64M1 model instead of the 32M1 for more program memory. They have the same datasheet and seem to be the same in every other way, just that the 64M1 has more program memory, EEPROM, and SRAM.

**Comment by alihaydaroglu** [comment]
this seems like a good idea... did we ever make a decision on it?

**Comment by brunofalmeida** [comment]
Yeah, it should just be a matter of updating the flags in the makefiles.

**Comment by hirolemo** [comment]
Are the 64M1s larger/smaller than the 32M1s. Has using them been decided?

**Comment by brunofalmeida** [comment]
We decided to use the 64M1. It's exactly the same physical size and footprint as the 32M1 so that doesn't matter for PCB layout.

**Comment by vogeldylan** [comment]
When the DigiKey order goes out I'll try to make sure the 64M1's are ordered.
